config/routes.rb

```diff
@@ -27,6 +27,8 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resources :health_care_application, only: [:create, :index]
```

Comment:
Just wanted to make sure that this is in fact desired behavior. 

A user can have more than 1 health care application. Is that correct?